pkg/serverless/invocationlifecycle/trace.go

```diff
@@ -65,3 +84,27 @@ func endExecutionSpan(processTrace func(p *api.Payload), endTime time.Time) {
 		TracerPayload: tracerPayload,
 	})
 }
+
+func convertRawPayload(rawPayload string) (invocationPayload, bool) {
```

Comment:
do we need to return two params? 
It seems that if the boolean is false, you ignore the first returned param. So maybe we could check only if the first param is nil or something like that?